api/server/volume.go

```diff
@@ -110,6 +111,49 @@ func (vd *volAPI) parseParam(r *http.Request, param string) (string, error) {
 	return "", fmt.Errorf("could not parse %s", param)
 }
 
+func (vd *volAPI) nodeIPtoIds(nodes []string) ([]string, error) {
```

Comment:
nit: can be `nodeIPtoIDs`

Reply:
i just put the code back, that's what it was before